pypit/scripts/coadd_1dspec.py

```diff
@@ -41,7 +41,7 @@ def main(args, unit_test=False):
     files = []
     for ifl in filelist:
         if '*' in ifl:
-            files.append(glob.glob(ifl))
```

Comment:
If you change the order of the elements of the `filenames` variable in the file `pypit/tests/files/coadd_UGC3672A_red.yaml` do you still get the correct result? My concern is when the wildcard is in the second filename (as in my example above). Similarly, if you have two filenames, both containing wildcards, I think the first will be ignored (i.e. the `files` variable is reset during the second pass through the for loop). I might be wrong (!!) but that's my understanding of what the code is doing.